dts/common/nordic/nrf54l09.dtsi

```diff
@@ -63,7 +65,8 @@
 		hfpll: hfpll {
 			compatible = "fixed-clock";
 			#clock-cells = <0>;
-			clock-frequency = <DT_FREQ_M(128)>;
+			/* NRFX-6881: Restore to 128MHz when PDK becomes available. */
```

Comment:
it's already done on board level: https://github.com/zephyrproject-rtos/zephyr/commit/d619fd607cbb3551b08dd6f3fa8796cd799f06ae